packages/react/src/index.ts

```diff
@@ -1,5 +1,5 @@
 export * from './components';
-export { useBreakpointValue, useTheme } from './hooks';
+export { useBreakpointValue, useTheme, useColorMode } from './hooks';
```

Comment:
Is this API currently public? If not, can we export via /internals?

Reply:
export it from here: https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/src/internal.ts